cmd/nodecmd/create_gcp.go

```diff
@@ -307,3 +305,11 @@ func updateClustersConfigGCPKeyFilepath(projectName, serviceAccountKeyFilepath s
 	}
 	return app.WriteClustersConfigFile(&clustersConfig)
 }
+
+func defaultAvalancheCLIPrefix() string {
+	usr, err := user.Current()
+	if err != nil {
+		return constants.AnsibleSSHUser + constants.AvalancheCLISuffix
```

Comment:
my thinking was that this is not critical path and we can afford to return some default and not stop cli because of it.
also if we return error in addition, we will have to check it and push it all up until we exit app because of it. 
Also I would not imagine scenario when `user.Current()` can actually return error, so wanted to avoid all this complication for something simple like returing a string

Reply:
You probably want to propagate errors

Reply:
k will fix